nixos/modules/services/misc/ringboard.nix

```diff
@@ -0,0 +1,79 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.ringboard;
+in
+{
+  options.services.ringboard = {
+    x11.enable = lib.mkEnableOption "X11 support for Ringboard";
+    wayland.enable = lib.mkEnableOption "Wayland support for Ringboard";
+    x11Package = lib.mkPackageOption pkgs "ringboard" { };
+    waylandPackage = lib.mkPackageOption pkgs "ringboard-wayland" { };
```

Comment:
Not a big deal, but it feels a bit weird to have `x11.enable` but not `x11.package`. I think it would be better to either have `x11.enable` and `x11.package` or `enableX11` and `x11Package` for consistency.